app/assets/javascripts/subNav.js

```diff
@@ -0,0 +1,29 @@
+(function(window) {
+    "use strict";
+
+    document.addEventListener('DOMContentLoaded', function() {
+        const sideNavLinks = document.querySelectorAll('.usa-sidenav__item');
+        const currentURL = window.location.pathname;
+
+        sideNavLinks.forEach(link => {
+            const sublist = link.querySelector('.usa-sidenav__sublist');
+            const parentLink = link.querySelector('a');
+
+            const isMatch = parentLink && sublist && currentURL === parentLink.getAttribute('href');
+            if (isMatch) {
+                sublist.classList.replace('display-none', 'display-block');
+            }
+
+            link.addEventListener('click', function(event) {
+                if (sublist && event.target === parentLink) {
+                    event.preventDefault();
```

Comment:
I'm assuming this is intentional; if so, is there any other default behavior we need it to resume doing or explicitly re-enable after the custom behavior?  I'm just double checking to be sure!